papis/commands/add.py

```diff
@@ -613,6 +618,11 @@ def cli(files: List[str],
                 "Select matching importers (for instance 0, 1, 3-10, a, all...)")
 
             matching_importers = [matching_importers[i] for i in matching_indices]
+        elif matching_importers:
+            if len(matching_importers) >= 2:
+                matching_importers = [matching_importers[1]]   
+            else:
+                matching_importers = [matching_importers[0]]
```

Comment:
From the previous pull request #613:

Can you explain the logic behind this? I'd expect that in batch mode, we just merge all matching importers.

Reply:
Hum, it is right we could just merge all the importers yes! I was thinking just select one (the 1.) but in fact it could be betetr to select all. Does it not have a bad side effect? A possibility of messing things? If we merge all the importers:
1) my last commit hacking utils.py to set the fallback downloader always first is useless
2) We can just remove this block of code you quote. So if batch, the matching_importers are not modified (all are selected).

Reply:
The system isn't quite that smart, so I'm not quite sure about the effect of merging data from multiple importers automatically. I haven't used it like that, but I would expect that
* Just a few importers match for a given query, e.g. giving a DOI won't match the other importers.
* The importers are generally pretty solid sources (as opposed to the downloaders), so even if they overwrite data, it should be relatively ok.